code/crates/test/proto/sync.proto

```diff
@@ -17,11 +17,12 @@ message Status {
 
 message ValueRequest {
     uint64 height = 1;
+    optional uint64 end_height = 2;
 }
 
 message ValueResponse {
-    uint64 height = 1;
-    SyncedValue value = 2;
+    uint64 start_height = 1;
+    repeated SyncedValue values = 2;
```

Comment:
We can safely add `repeated` and backward compatibility because an old node expecting a single value will parse the last occurrence of the field and ignore the previous ones.